### DISCUSSION TITLE
SHA value of the Safe Exam Browser

### DISCUSSION THREAD

**Discussion by SalamShahid** [SafeExamBrowser/seb-win-refactoring#940] [General]
Would you be able to provide the SHA value of the Safe Exam Browser application so that we may whitelist it in advance in our EDR software?

Thanks
Shahid

**Comment by dbuechel** [comment]
Could you please specify what exactly you mean with "the SHA value of the Safe Exam Browser application"? The SHA1 hash value of the setup can be consulted on the download page, see https://safeexambrowser.org/download_en.html.
